src/inc/gcinfotypes.h

```diff
@@ -240,6 +240,25 @@ inline bool IsValidReturnKind(ReturnKind returnKind)
         ;
 }
 
+inline bool IsGCReturnKind(ReturnKind returnKind)
+{
+    switch (returnKind)
+    {
+    case RT_Object:
```

Comment:
RT_Byref case is also handled, because the code treats all gc-returns as byrefs conservatively.
https://github.com/dotnet/coreclr/blob/master/src/vm/gccover.cpp#L1742 
However, two-reg returns are not handled correctly; so I filed #8665.